src/features/jserrors/instrument/index.js

```diff
@@ -71,67 +62,55 @@ export class Instrument extends InstrumentBase {
   /** Restoration and resource release tasks to be done if JS error loader is being aborted. Unwind changes to globals. */
   #abort () {
     this.removeOnAbort?.abort()
+    this.#seenErrors.clear()
     this.abortHandler = undefined // weakly allow this abort op to run only once
   }
 
+  #castError (error) {
+    if (error instanceof Error) {
+      return error
+    }
+
+    if (typeof error.message !== 'undefined') {
+      return new UncaughtError(error.message, error.filename, error.lineno, error.colno)
+    }
```

Comment:
Is there a case when this is called with an error that's not an Error instance yet has a message?
I see `fn-err` and all `internal-error` events emitted with an actual error object from try-catch.

Reply:
`throw` can be passed any value.